metriq_gym/cli.py

```diff
@@ -68,6 +68,12 @@ def parse_arguments() -> argparse.Namespace:
     poll_parser = subparsers.add_parser("poll", help="Poll jobs")
     poll_parser.add_argument("--job_id", type=str, required=False, help="Job ID to poll (optional)")
 
+    # Subparser for upload.
+    upload_parser = subparsers.add_parser("upload", help="Upload results to Metriq")
+    upload_parser.add_argument("--job_id", type=str, required=True, help="Job ID to upload")
+    upload_parser.add_argument("-s", "--submission_id", type=int, required=True, help="Metriq submission ID (needed to upload results)")
+    upload_parser.add_argument("-t", "--token", type=str, required=True, help="Metriq API token (needed to upload results)")
```

Comment:
Let's have the token as an env variable in the .env file instead, with the name of the variable `METRIQ_API_TOKEN` specified in the `.env.example` file. This is the pattern we have adopted so far for authentication, and it's working well.

Reply:
Done.